src/main/java/org/commonjava/service/promote/tracking/cassandra/SchemaUtils.java

```diff
@@ -35,4 +37,16 @@ public static String getSchemaCreateTableTracking( String keySpace )
                 + "PRIMARY KEY (trackingId, promotionId)"
                 + ");";
     }
+
+    public static String getSchemaCreateTableQueryByPath( String keySpace )
```

Comment:
I remembered that all these things are hosed in path-mapped-storage, so should we include this there or keeping here works ?

Reply:
as we spread functions to each service, table schema are with their own service. The promote, tracking, and repo service all have their own table schema within own code.

Reply:
yeah, I see. LGTM.